src/components/scanHostList/scanHostList.js

```diff
@@ -88,7 +88,7 @@ class ScanHostList extends React.Component {
 
     return (
       <div className="quipucords-infinite-results">
-        <Grid fluid onScroll={this.onScrollList} className="quipucords-infinite-list">
+        <Grid onScroll={this.onScrollList} className="container-fluid quipucords-infinite-list">
```

Comment:
required:
- lets remove this change from this PR ... reason it's actually a wrapper around Grid... so grid inside grid inside grid... very meta
- OR we can attempt to include the `Grid` changes for the files that call this component, `ScanListItem.js` and `ScanHostList` ... but since we plan on potentially using Grid updates as a clean up task after parent components... it may be easier to just forgo this change for now

But If we keep this file the changes required are...
- same comment about converting the Grid to `<List isPlain` and `<ListItem.`
- throw in the optional chain on `hostsList?.map` for consistency

question:
- possibly apply a similar but NEW `max-height` css class to all lists in this PR, dependent on question to @bclarhk

Reply:
Thank you for your helpful feedback here - pushed some updates to the files you mentioned. Let me know if this is what you had in mind... might not be _exactly_ 1:1 since the `icon` prop on PF4 `List` has different styling now!